packages/go/analysis/ad/adcs.go

```diff
@@ -226,6 +227,8 @@ func postADCSPreProcessStep1(ctx context.Context, db graph.Database, enterpriseC
 		return &analysis.AtomicPostProcessingStats{}, fmt.Errorf("failed post processing for %s: %w", ad.EnterpriseCAFor.String(), err)
 	} else if err = PostCanAbuseUPNCertMapping(ctx, db, operation, enterpriseCertAuthorities); err != nil {
 		return &analysis.AtomicPostProcessingStats{}, fmt.Errorf("failed post processing for %s: %w", ad.CanAbuseUPNCertMapping.String(), err)
+	} else if err = PostCanAbuseWeakCertBinding(ctx, db, operation, enterpriseCertAuthorities); err != nil {
+		return &analysis.AtomicPostProcessingStats{}, fmt.Errorf("failed post processing for %s: %w", ad.CanAbuseWeakCertBinding.String(), err)
```

Comment:
A call to `operation.Done()` should be added before the return in these two conditionals similar to what was done in [this PR](https://github.com/SpecterOps/BloodHound/pull/305/files) so that the connection doesn't hang when an error is encountered.